src/Discord.Net.Core/Utils/Comparers.cs

```diff
@@ -46,7 +40,7 @@ public override bool Equals(TEntity x, TEntity y)
                     (null, null) => true,
                     (null, _)    => false,
                     (_, null)    => false,
```

Comment:
Have we updated to `<LangVersion>9.0</LangVersion>` yet? If yes, these two cases could be condensed with the `or` pattern operator. (Just thinking out loud.)